lib/cisco_node_utils/fabricpath_topology.rb

```diff
@@ -72,54 +67,47 @@ def fabricpath_feature_set(fabricpath_set)
       FabricpathGlobal.fabricpath_feature_set(fabricpath_set)
     end
 
-    def state
-      result = config_get('fabricpath_topology', 'state', @topo_id)
-      return default_state if result.nil?
-      case result
-      when /Up/
-        return 'up'
-      when /Down/
-        return 'default'
+    def member_vlans
+      config_get('fabricpath_topology', 'member_vlans',
+                 @topo_id).gsub(/\s+/, '')
```

Comment:
Yes, I was thinking that some platform might introduce a space in between the vlan range. But from n7k, cli output, it doesn't look like it does:

fabricpath topology 10
  member vlan **10-20,25-30**

The range_summarize() puppet layer function gets rid of all the spaces. In order to get a reliable output which can match with that, I am getting rid of all the spaces if they exist